### PR TITLE
Add guidance on when having multiple suppliers per award

### LINKED ISSUE #912: Guidance: Explain when to have multiple suppliers per award
Initial ideas:

* Consortia, where multiple suppliers put together a single bid, and are awarded as a group. In some countries, these suppliers only create a legal entity after the award, to sign the contract; or, they decide on a prime contractor, with the others being subcontractors. #370 #142

Noting that the current model for framework agreements has multiple suppliers (it stores the 'multi-use list' as suppliers on an award #909), though this might change per #440 and #909.

### PR DESCRIPTION
**Pull request by yolile**
closes #912

### CODE REVIEW

**Comment by yolile** [comment]
@duncandewhurst this is ready to review but I will include the link to the framework example too when #1108 is merged

File: schema/release-schema.json [hunk 1]
```diff
@@ -549,7 +549,7 @@
         },
         "suppliers": {
           "title": "Suppliers",
-          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks.",
+          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks. There can be multiple suppliers for a single award if, for example, the award recipient is an unincorporated consortium or if the award represents a framework agreement set up.",
```
Comment (duncandewhurst) [inline schema/release-schema.json]: Aligning with the language in the frameworks example:
```suggestion
          "description": "The suppliers awarded this award. If different suppliers have been awarded different items or values, these should be split into separate award blocks. There can be multiple suppliers for a single award if, for example, the award recipient is an unincorporated consortium or if the award represents the addition of suppliers as parties to a framework agreement.",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

**Comment by jpmckinney** [comment]
@yolile Is this still draft?

**Comment by yolile** [comment]
@jpmckinney this depends on #1123 to add the link to the new FA documentation

**Comment by jpmckinney** [comment]
I think we can merge this first, and just add the link in #1123.

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**